### PR TITLE
Second section

### PR DESCRIPTION
**Pull request by AngelinaPtrv**

### CODE REVIEW

File: style/fonts.css [hunk 1]
```diff
@@ -22,4 +22,10 @@
   font-family: Gotham Pro Light;
   src: url("./GothamPro/GothamPro-Light.woff2") format("woff2");
   src: url("./GothamPro/GothamPro-Light.woff") format("woff");
-}
+}
```
Comment (seasmail) [inline style/fonts.css:25]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

File: style/styles_footer.css [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+  flex-direction: column;
+}
+
+
+.footer-sign__button {
+  background-color: #000;
+  color: #fff;
+  width: 136.57px;
+  height: 29.7px;
+  font-family: "Gotham Pro Bold";
```
Comment (seasmail) [inline style/styles_footer.css:39]: Codacy Issue found: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_stories.css [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+@media (max-width: 320px) {
+  .stories__block {
+    padding-left: 12px;
+  }
+
+  .stories__title {
+    padding-left: 0;
+  }
+}
```
Comment (seasmail) [inline style/styles_stories.css:267]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_social.css [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+  display: none;
+}
+
+.social__pic {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+ @media (max-width: 1100px) {
```
Comment (seasmail) [inline style/styles_social.css:31]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_footer.css [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+  margin: 0;
+  padding-bottom: 10px;
+}
+
+.footer-navigation {
+  display: flex;
+  flex-direction: column;
+}
+
+
```
Comment (seasmail) [inline style/styles_footer.css:33]: Codacy Issue found: [Expected no more than 1 empty line (max-empty-lines)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_footer.css [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+  }
+}
+
+@media (max-width: 640px) {
+  .footer-sign {
+    padding-top: 30px;
+  }
+}
+
+@media(max-width: 454px) {
```
Comment (seasmail) [inline style/styles_footer.css:216]: Codacy Issue found: [Expected single space after at-rule name "@media" (at-rule-name-space-after)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_footer.css [hunk 7]
```diff
@@ -0,0 +1,363 @@
...
+  .facebook {
+    width: 6.92px;
+    height: 13.83;
+  }
+
+  .twitter {
+    width: 15.56px;
+    height: 12.64px;
+  }
+
```
Comment (seasmail) [inline style/styles_footer.css:174]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_footer.css [hunk 8]
```diff
@@ -0,0 +1,363 @@
...
+  .navigation_three {
+    margin-right: 0;
+  }
+
+  .footer__block {
+    padding-left: 0;
+  }
+
+  .footer-sign {
```
Comment (seasmail) [inline style/styles_footer.css:359]: Codacy Issue found: [Unexpected duplicate selector ".footer-sign", first used at line 339 (no-duplicate-selectors)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/fonts.css (hunk 1 above)
Comment (seasmail) [inline style/fonts.css:25]: Codacy Issue found: [Expected newline after "}" (block-closing-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_stories.css [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+    width: 312px;
+    height: 166px;
+    padding-bottom: 28px;
+  }
+
+  .stories-item__title {
+    font-size: 25px;
+    padding-bottom: 8px;
+  }
+
```
Comment (seasmail) [inline style/styles_stories.css:162]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_footer.css [hunk 10]
```diff
@@ -0,0 +1,363 @@
...
+    font-size: 13px;
+  }
+}
+
+@media (max-width: 320px) {
+  .footer {
+    width: 313px;
+    margin: 0 auto;
+  }
+  .footer-sign {
```
Comment (seasmail) [inline style/styles_footer.css:339]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_learn.css [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+  content: url("../pic/Arrow.png");
+  padding-left: 8px;
+}
+
+.learn__video {
+  padding: 68px 0 66px;
+  margin-right: 8px;
+}
+
+@media(max-width: 1100px) {
```
Comment (seasmail) [inline style/styles_learn.css:50]: Codacy Issue found: [Expected single space after at-rule name "@media" (at-rule-name-space-after)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

File: style/fonts.css [hunk 12]
```diff
@@ -23,3 +23,9 @@
   src: url("./GothamPro/GothamPro-Light.woff2") format("woff2");
   src: url("./GothamPro/GothamPro-Light.woff") format("woff");
 }
+
+@font-face {
+  font-family: Gotham Pro Regular;
+  src: url("./GothamPro/GothamProRegular.woff2") format("woff2");
+  src: url("./GothamPro/GothamProRegular.woff") format("woff");
+}
```
Comment (seasmail) [inline style/fonts.css:31]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_footer.css [hunk 13]
```diff
@@ -0,0 +1,363 @@
...
+  justify-content: space-between;
+  align-items: center;
+  width: 137px;
+  padding-top: 34px;
+}
+
+.copyright {
+  font-family: "Gotham Pro Regular", sans-serif;
+  font-size: 12px;
+  padding-left: 484px;
```
Comment (vylenne) [inline style/styles_footer.css]: так пытаться центрировать не нужно. замени этот паддинг на text-align

File: style/styles_footer.css [hunk 14]
```diff
@@ -0,0 +1,363 @@
+.footer {
+  background-color: #000;
+  color: #fff;
+  width: 100%;
+}
+
+.footer__block {
+  display: flex;
+  padding-top: 70px;
+  padding-left: 300px;
```
Comment (vylenne) [inline style/styles_footer.css]: тут тоже почему пытаешься паддингом выравнивать? :) центрируй так же флексами или другими известными тебе способами ;)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

File: style/styles_footer.css [hunk 15]
```diff
@@ -0,0 +1,298 @@
...
+    padding-top: 60px;
+  }
+
+  .footer-navigation__title {
+    white-space: nowrap;
+    padding-bottom: 7px;
+  }
+
+  .navigation_one {
+    flex-basis: 141.23px
```
Comment (seasmail) [inline style/styles_footer.css:112]: Codacy Issue found: [Expected a trailing semicolon (declaration-block-trailing-semicolon)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_footer.css [hunk 16]
```diff
@@ -0,0 +1,298 @@
...
+  background-color: #000;
+  color: #fff;
+  width: 100%;
+}
+
+.footer__block {
+  display: flex;
+  padding-top: 70px;
+  justify-content: center;
+  /* padding-left: 300px; */
```
Comment (seasmail) [inline style/styles_footer.css:11]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_social.css [hunk 17]
```diff
@@ -0,0 +1,213 @@
...
+  .social-seven {
+    display: none;
+  }
+
+  .social-six {
+    display: none;
+  }
+}
+
+
```
Comment (seasmail) [inline style/styles_social.css:138]: Codacy Issue found: [Expected no more than 1 empty line (max-empty-lines)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

File: style/styles_social.css [hunk 18]
```diff
@@ -0,0 +1,145 @@
...
+    padding-top: 90px;
+    padding-bottom: 82px;
+  }
+
+  .social-seven {
+    display: inline;
+  }
+}
+
+
```
Comment (seasmail) [inline style/styles_social.css:45]: Codacy Issue found: [Expected no more than 1 empty line (max-empty-lines)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

File: style/styles_learn.css [hunk 19]
```diff
@@ -0,0 +1,214 @@
...
+    margin: 0 0 10px;
+    line-height: 16.4px;
+  }
+
+  .learn__block-link::after {
+    padding-left: 13px;
+  }
+}
+
+ @media (max-width: 768px) {
```
Comment (seasmail) [inline style/styles_learn.css:95]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_learn.css [hunk 20]
```diff
@@ -0,0 +1,214 @@
...
+
+@media (max-width: 375px) {
+  .learn__video {
+    width: 285px;
+  }
+
+  .learn__block-text {
+    width: 100%;
+  }
+  .learn__block-title {
```
Comment (seasmail) [inline style/styles_learn.css:197]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

File: style/styles_learn.css (hunk 19 above)
Comment (seasmail) [inline style/styles_learn.css:95]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_learn.css (hunk 20 above)
Comment (seasmail) [inline style/styles_learn.css:197]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_stories.css [hunk 21]
```diff
@@ -0,0 +1,260 @@
...
+  .stories-item__title {
+    font-size: 20px;
+    padding-bottom: 10px;
+  }
+
+  .item-first .stories-item__article {
+    width: 100%;
+  }
+  .stories-item__article {
```
Comment (seasmail) [inline style/styles_stories.css:239]: Codacy Issue found: [Expected selector ".stories-item__article" to come before selector ".item-first .stories-item__article" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

Comment (seasmail) [inline style/styles_stories.css:239]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

**Comment by seasmail** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582&bid=18465664)

File: style/styles_stories.css [hunk 22]
```diff
@@ -0,0 +1,261 @@
...
+    font-size: 14px;
+    line-height: 17.7px;
+    padding-bottom: 26px;
+    margin: 0;
+  }
+
+  .item-first .stories-item__article {
+    width: 100%;
+  }
+
```
Comment (seasmail) [inline style/styles_stories.css:246]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

File: style/styles_learn.css [hunk 23]
```diff
@@ -0,0 +1,215 @@
...
+
+@media (max-width: 375px) {
+  .learn__video {
+    width: 285px;
+  }
+
+  .learn__block-text {
+    width: 100%;
+  }
+
```
Comment (seasmail) [inline style/styles_learn.css:197]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)

**Pull request merged**